### PR TITLE
[fix](load) fix memtable agg functions

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes
fix init agg functions in memtable.

### CODE REVIEW

**Comment by liaoxin01** [comment]
run buildall

File: be/src/olap/memtable.cpp [hunk 1]
```diff
@@ -187,7 +187,7 @@ Status MemTable::insert(const vectorized::Block* input_block,
         _vec_row_comparator->set_block(&_input_mutable_block);
         _output_mutable_block = vectorized::MutableBlock::build_mutable_block(&cloneBlock);
         if (_keys_type != KeysType::DUP_KEYS) {
-            _init_agg_functions(input_block);
+            _init_agg_functions(&cloneBlock);
```
Comment (yiguolei) [inline be/src/olap/memtable.cpp:190]: add comment here, to explain why cloned block not input block

File: be/src/olap/memtable.cpp [hunk 2]
```diff
@@ -187,6 +187,8 @@ Status MemTable::insert(const vectorized::Block* input_block,
         _vec_row_comparator->set_block(&_input_mutable_block);
         _output_mutable_block = vectorized::MutableBlock::build_mutable_block(&cloneBlock);
         if (_keys_type != KeysType::DUP_KEYS) {
+            // there may be additional intermediate columns in input_block
+            // we only need columns indicated by column offset in the output 
             _init_agg_functions(&cloneBlock);
```
Comment (liaoxin01) [inline be/src/olap/memtable.cpp:192]: cloneBlock -> clone_block

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**